### DISCUSSION TITLE
Need Example in Python to publish on topic /cmd_audio, can't get to append a Note to the list notes. Thanks.

### DISCUSSION THREAD

**Discussion by hiroshiuema** [iRobotEducation/create3_docs#605] [Questions]
### How are you connecting to your Create 3?

Wi-Fi (ROS 2)

### Computer(s) Model(s) and Operating System(s)

Dell, i5 processor, running Ubuntu 22.04

### Which version of ROS 2 is installed on your computer?

Humble

### Which firmware version is installed on your robot?

G31

### Which RMW is your robot running?

I don't know

### Is the robot connected to a network? If so, what is the network type?

yes, router with/without internet access, isolated only for testing turtlebot 4 and other educational platforms

### Are there multiple Create 3 robots connected to your network?

No

### Is multicast enabled?

Yes

### What is the Adapter Board's USB/BLE Toggle currently switched to?

USB

### Describe your question.

I couldn't find any Python examples on how to publish on the  /cmd_audio topic. At the begining I couldn't append to the notes list, but once I could append, I couldn't publish. Is there any publisher example for this topic?

Thanks.

**Comment by alsora** [comment]
I don't think there are any examples for that.

The `/cmd_audio` topic has nothing special, so I recommend you to look into how to write a standard ROS 2 publisher using Python.
https://docs.ros.org/en/galactic/Tutorials/Beginner-Client-Libraries/Writing-A-Simple-Py-Publisher-And-Subscriber.html

After you try that, if you still have issues, please provide more details about exactly what are you doing (i.e. provide code) and what's not working.
